### ISSUE TITLE
Add a poll component

### ISSUE THREAD

**Issue by TechWatching** [slidevjs/slidev#477]
**Describe the solution you'd like**
[DeckDeckGo](https://github.com/deckgo/deckdeckgo) has an interesting feature which is the ability to add polls to a slide with a place where people could vote. I don't know if it could be something possible with slidev but it seems very interesting.

**Comment by antfu** [comment]
Embedding an iframe with the polling service you prefer would be the best way to match preference and be independent IMO

**Comment by tonai** [comment]
Hello @TechWatching ,
I created a little library for creating polls inside slidev: https://github.com/Smile-SA/slidev-component-poll
Maybe you can be interested.

**Comment by TechWatching** [comment]
> Hello @TechWatching , I created a little library for creating polls inside slidev: https://github.com/Smile-SA/slidev-component-poll Maybe you can be interested.

It seems really cool. But I have not been able to make it work:
```
Error: Build failed with 5 errors:
node_modules/slidev-component-poll/dist/poll.mjs:2:47: ERROR: Could not read from file: /@fs/home/projects/slidev-b5halc/node_modules/@slidev/client/logic/nav
node_modules/slidev-component-poll/dist/poll.mjs:3:20: ERROR: Could not resolve "../../../../../../@slidev/configs"
```

**Comment by tonai** [comment]
Thank you for your quick test.
I fixed some issues related to these errors, but there is still some of them.
I don't get those errors because it works well when you import the components locally but not when you import them from a released package. Need to figure why...

A quick and dirty workaround is to install the components locally on your project (and using latest version 0.1.2):
```bash
mkdir components
cd components
ln -s ../node_modules/slidev-component-poll/src/* .
```

Also I updated the [README](https://github.com/Smile-SA/slidev-component-poll) because you also need to create a `vite.config.ts` file with the given configuration.
